cluster/addons/dns/skydns-rc.yaml.base

```diff
@@ -103,9 +101,7 @@ spec:
       - name: healthz
         image: gcr.io/google_containers/exechealthz-amd64:1.1
         resources:
-          # keep request = limit to keep this container in guaranteed class
           limits:
-            cpu: 10m
```

Comment:
Adding more TODO seems to be making the yaml outputs incompatible. Given the time constraint, I chose to not do that. I'm happy to post another PR tomorrow once I have more time to figure out why the yaml outputs were broken when TODOs are being added.